src/ProgressOnderwijsUtils/Collections/MaybeExtensions.cs

```diff
@@ -25,7 +25,7 @@ public static class MaybeExtensions
 
         [CanBeNull]
         [Pure]
-        public static TOk ValueOrNull<TOk, TError>(this Maybe<TOk, TError> state)
+        public static TOk? ValueOrNull<TOk, TError>(this Maybe<TOk, TError> state)
             where TOk : class

```

Comment:
zoals irl besproken zou het kunnen zijn dat deze annotaties te strak zijn en problemen gaan veroorzaken mits we hier maybes-om-nullalbles hebben.